ports/stb/portfile.cmake

```
@@ -1,11 +1,57 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO nothings/stb
-    REF b42009b3b9d4ca35bc703f5310eedc74f584be58 # accessed on 2020-09-14
-    SHA512 a9ba80d19dae4e527171bb02e1caa4d3eb0704cdf7f8fef1a7a09e6b05c65b829b0aa580f469f158a39bf13018289f40c1680ab5c22bfa5e932bff94eced475d
+    REF 80c8f6af0304588b9d780a41015472013b705194 # accessed on 2021-06-24
```

```suggestion
    REF 80c8f6af0304588b9d780a41015472013b705194 # accessed on 2021-06-26
```
The date should be consistent with that in vcpkg.json.